usagov/vote-gov-drupal
# Vote-1617 -> update to external links tests

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1617](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1617&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR is to fix the external links tests that were found to be out of date after running post sprint 35

> **Note**
>I have manually checked each link that was added and can verify that they are working, I have added them to the excluded >links list for both English and Spanish.  How ever I do think it is worth looking into either how to better write these tests or >possibly looking into another tool to do our link validation. 
>
>I have created that ticket here [Vote-1651](https://bixal-projects.atlassian.net/browse/VOTE-1651)

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. start local 
3. cd into `testing`
4. run`npm run cy:proofer` verify test is passing 
5. run `npm run cy:open` and select the `excluded-links.cy.js` file and verify that test is passing
